src/util.ts

```diff
@@ -98,12 +98,17 @@ export function canonicalizeGOPATHPrefix(filename: string): string {
 	if (!gopath) return filename;
 	let workspaces = gopath.split(path.delimiter);
 	let filenameLowercase = filename.toLowerCase();
+
```

Comment:
Can you add a comment here about the logic being used, so that the next person knows whats going on?
The unit tests are failing in travis due to linting issues. Run `npm run lint` locally and fix the reported errors.